test_schemas/en/test_list_collector_driving_checkbox.json

```diff
@@ -670,6 +670,11 @@
                             "id": "who-lives-here-summary",
                             "title": "Who Lives Here Summary",
                             "type": "SectionSummary"
+                        },
+                        {
+                            "id": "summary",
```

Comment:
Don't think this will work as expected when you merge it, due to the changes made in SectionSummary. Summary needs to be in a separate section as the section summary will always be the last block if its enabled, so this might go Block -> summary -> section summary